--- NODELIST.093	1998-04-05 03:34:18.000000000 +0800
+++ NODELIST.100	1998-04-10 08:23:16.000000000 +0800
@@ -1,4 +1,4 @@
-;A Zone Nodelist for Friday, March 27, 1998 -- Day number 086 : 43182
+;A Zone Nodelist for Friday, April 10, 1998 -- Day number 100 : 42333
 ;A
 ;S A listing of all the systems within CFidonet
 ;S
@@ -92,7 +92,7 @@
 Hub,2000,shanghai_hub,ShangHai,Wang_Quanhua,86-21-6325-7546,9600,CM,XA,V32B,V42B
 ;
 Host,655,Guangdong_Net,Shenzhen,Lester_Lee,86-755-667-2764,9600,CM,V34,CSP,HST
-Hub,100,ShenZhen_Hub,Shenzhen,Lester_Lee,86-755-667-2764,9600,CM,V34,CSP,HST
+Hub,100,ShenZhen_Hub,Shenzhen,Lester_Lee,86-755-667-2764,9600,CM,V34,V42B
 ,101,Ponysoft_BBS,Shenzhen,Pony_Ma,86-755-559-9961,9600,CM,XA,V32B,V32T,V42B
 ,104,OrientExpress_BBS,Shenzhen,Daniel_Lee,86-755-686-2299,9600,CM,XA,V32B,VFC,V34
 ,105,Colorful_Life,Shenzhen,Chen_Zhi_Jian,86-755-380-5152,9600,CM,XX,V32B,VFC,V34
@@ -127,7 +127,7 @@
 ,901,Gold_Coast,Lufeng,Chen_Song,86-660-882-6662,9600,CM,XA,V42B,V32B,V34
 ,1000,XinHui_Hub,Xin_Hui,Huang_Yongkai,86-750-647-7888,9600,CM,XA,V32B,V32T,V42B
 ,1001,InterLink_BBS,Xin_Hui,Huang_Yongkai,86-750-647-7888,9600,CM,XA,V32B,V32T,V42B
-,1002,Sun_Link_BBS,Xin_Hui,Jing_Jun,86-750-666-2878,9600,CM,XA,V32B,V32T,V42B
+,1002,Sun_Link_BBS,Xin_Hui,Jing_Jun,86-750-666-2878,9600,CM,XA,V34,V32B,V42B
 ;
 Host,656,SouthwestChina_Net,Chengdu,Rene_Ren,86-28-774-5862,9600,MO,TAH,V34b,V42b
 Down,1,CDWM_BBS,Chengdu,Rene_Ren,86-28-554-5031,9600,CM,V34,V42B,XA
